cli/Cargo.toml

```diff
@@ -22,9 +22,9 @@ atty = "0.2"
 base64 = "0.12.2"
 simplelog = "0.8.0"
 log = "0.4.8"
-squawk-parser = { version = "0.7.0", path = "../parser" }
-squawk-linter = { version = "0.7.0", path = "../linter" }
-squawk-github = { version = "0.7.0", path = "../github" }
+squawk-parser = { version = "0.0.0", path = "../parser" }
```

Comment:
stole this idea from rust-analyzer

Reply:
https://github.com/rust-analyzer/rust-analyzer/blob/04f03a360ab8fef3d9c0ff84de2d39b8a196c717/crates/ide/Cargo.toml#L25-L34